### ISSUE TITLE
java.lang.IllegalStateException: No match found

### ISSUE THREAD

**Issue by lehphyro** [MeltwaterArchive/dropwizard-extra#13]
Hi,

I got this with exception:

```
java.lang.IllegalStateException: No match found
    at java.util.regex.Matcher.group(Matcher.java:485) ~[na:1.7.0_10]
    at com.datasift.dropwizard.curator.ensemble.DropwizardConfiguredZooKeeperFactory$DynamicZooKeeperConfiguration.<init>(DropwizardConfiguredZooKeeperFactory.java:60) ~[dropwizard-extra-curator-0.6.2-1.jar:na]
    at com.datasift.dropwizard.curator.ensemble.DropwizardConfiguredZooKeeperFactory.newZooKeeper(DropwizardConfiguredZooKeeperFactory.java:44) ~[dropwizard-extra-curator-0.6.2-1.jar:na]
    at com.netflix.curator.HandleHolder$1.getZooKeeper(HandleHolder.java:93) ~[curator-client-1.2.4.jar:na]
    at com.netflix.curator.HandleHolder.getZooKeeper(HandleHolder.java:54) ~[curator-client-1.2.4.jar:na]
    at com.netflix.curator.ConnectionState.reset(ConnectionState.java:155) ~[curator-client-1.2.4.jar:na]
    at com.netflix.curator.ConnectionState.start(ConnectionState.java:117) ~[curator-client-1.2.4.jar:na]
    at com.netflix.curator.CuratorZookeeperClient.start(CuratorZookeeperClient.java:182) ~[curator-client-1.2.4.jar:na]
    at com.netflix.curator.framework.imps.CuratorFrameworkImpl.start(CuratorFrameworkImpl.java:200) ~[curator-framework-1.2.4.jar:na]
    at com.datasift.dropwizard.curator.ManagedCuratorFramework.start(ManagedCuratorFramework.java:24) [dropwizard-extra-curator-0.6.2-1.jar:na]
    at com.yammer.dropwizard.config.Environment.doStart(Environment.java:116) [dropwizard-core-0.6.2.jar:na] [4 skipped]
    at com.yammer.dropwizard.cli.ServerCommand.run(ServerCommand.java:48) [dropwizard-core-0.6.2.jar:na] [6 skipped]
    at com.yammer.dropwizard.cli.EnvironmentCommand.run(EnvironmentCommand.java:39) [dropwizard-core-0.6.2.jar:na]
    at com.yammer.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:58) [dropwizard-core-0.6.2.jar:na]
    at com.yammer.dropwizard.cli.Cli.run(Cli.java:53) [dropwizard-core-0.6.2.jar:na]
    at com.yammer.dropwizard.Service.run(Service.java:61) [dropwizard-core-0.6.2.jar:na]
    at com.example.helloworld.HelloWorldService.main(HelloWorldService.java:14) [classes/:na]
```

With this configuration:

```
curator:
  maxRetries: 3
  ensemble:
    namespace: testes
    hosts: [ 127.0.0.1 ]
```

Using com.datasift.dropwizard:dropwizard-extra-curator:0.6.2-1

Did I do something wrong in the configuration?

**Comment by nicktelford** [comment]
Sorry about that, thanks for reporting the bug.

The fix for this will be in both 0.7.0-1 and 0.6.2-2. I'll try to cut a release of 0.6.2-2 soon, as there's no work-around for this bug.

**Comment by lehphyro** [comment]
Thank you, do you plan to release 0.6.2-2 soon or you'll just wait and release the fix in version 0.7.0-1 only?

**Comment by nicktelford** [comment]
I've pushed 0.6.2-2 to central with this fix. There are no other changes.
